docs/netbox-hosting/netbox-plugins/netbox-cloud-pilot/Manage NetBox Settings.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Manage NetBox Settings
+slug: manage-netbox-settings
+sidebar_position: 4
+---
+
+After configuring your NetBox Cloud Pilot plugin, you can manage your NetBox settings directly through the VAP interface.
+
+## Managing Settings
+
+1. In your NetBox dashboard, go to the **NetBox Cloud Pilot** section.
+2. Select **Manage** and click on **Edit NetBox Settings** to access the settings form.
+
+<div style={{
+    display:'flex',
+    justifyContent: 'center',
+    margin: '0 0 1rem 0'
+}}>
+
+![Locale Dropdown](./img/ManageNetBoxSettings/img-01.png)
+
+</div>
+
+This form allows you to configure all NetBox settings that are supported for your version. You can modify settings such as Allowed Hosts, Database, Redis, System, Security and lots more.
+
+<div style={{
+    display:'flex',
+    justifyContent: 'center',
+    margin: '0 0 1rem 0'
+}}>
+
+![Locale Dropdown](./img/ManageNetBoxSettings/img-02.png)
+
+</div>
+
+:::danger Important Note
+
+- Ensure that at least one worker is running for the "default" queue before making changes. This is necessary because modifying settings will require a restart of all NetBox containers which is trigger in a background task.
```

Comment:
```suggestion
- Ensure that at least one worker is running for the "default" queue before making changes. This is necessary because modifying settings will require a restart of all NetBox containers which is triggered in a background task.
```